runtime/src/main/cpp/KString.cpp

```diff
@@ -671,17 +671,17 @@ int iswalnum_Konan(KChar ch) {
 }
 
 int iswspace_Konan(KChar ch) {
```

Comment:
Kotlin stdlib is also behaves differently than JDK: https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/src/kotlin/text/CharJVM.kt#L77